Update old readmes (#1979)
OSC/ondemand: apps/myjobs/README.md

# Job Composer (renamed from My Jobs)

> This app is a Rails app for Open OnDemandfor creating and managing batch jobs

Reviewer: Space was missing and the wording might be cleaner as something like:
"This is a Rails app in Open OnDemand for creating aand managing batch jobs..."

> This app is a Rails app for Open OnDemandfor creating and managing batch jobs

Reviewer: Space was missing and the wording might be cleaner as something like:
```
This is a Rails app in Open OnDemand for creating aand managing batch jobs...
```

Reviewer 2: make sure you change aand to and

Reviewer 2: Not sure what happened to Travis's remark, but this is what he had.

"This is a Rails app in Open OnDemand for creating and managing batch jobs..."

Author: ```suggestion
This app is a Rails app we ship with Open OnDemand that used to create and manage batch jobs
```

> from template directories. Like all Open OnDemand apps, it's meant to runs as a non-root user.

Reviewer 2: ```suggestion
from template directories. Like all Open OnDemand apps, it's meant to run as a non-root user.
```

This is a guide for developing this aplication and assumes you have nodejs & ruby available on the system.

You can find [template documentation here](https://osc.github.io/ood-documentation/latest/customization.html#custom-job-composer-templates).

## Developing

First, you'll need to clone this repo and make a symlink if you haven't don so already.

```text
mkdir -p ~/ondemand/dev
git clone https://github.com/OSC/ondemand.git ~/ondemand/src
cd ~/ondemand/dev
ln -s ../src/apps/myjobs
```

> Now build run `bin/setup` from within this directory to fetch all the dependencies

Reviewer: Now build run reads strange, maybe:
"Now run `bin/setup` from within..."

> and compile.

Reviewer: Now build run reads strange, maybe:
```
Now run `bin/setup` from within this directory to fetch all the dependencies
and compile.
```

Author: ```suggestion
Now run `bin/setup` from within this directory to fetch all the dependencies
and compile.
```
